src/compiler/diagnosticMessages.json

```diff
@@ -1311,7 +1311,7 @@
         "category": "Error",
         "code": 2427
     },
-    "All declarations of an interface must have identical type parameters.": {
+    "All declarations must have identical type parameters.": {

```

Comment:
This error message is only vaguely helpful at a glance. All class and interface declarations are named unless they are default exports (in which case they shouldn't get merged anyway), so let's use the name in the message.

```json
"All declarations of '{0}' must have identical type parameters."
```